docs/lms_config.ini

```diff
@@ -7,7 +7,7 @@ version = latest
 
 # below are the pdf settings for the pdf file
 [pdf_settings]
-pdf_base = https://media.readthedocs.io/pdf/open-edx-learner-guide
+pdf_base = https://media.readthedocs.org/pdf/open-edx-learner-guide
```

Comment:
Just putting a note here to match what we talked about in person. This file has openedx versions of the files. The edX versions are set in edX configuration (though presently there is not a way to set the PDF edX version, since the base_url actually differs from the HTML edX version-- this code is not being used though).